### PR TITLE
Alter filtering and ordering of order items

### LINKED ISSUE #369: DjangoAdmin table OrderItems should not be filterable by Order
We have 11300 orders in the system currently, and to get to the bottom of the page where the pagination is I have to scroll past these

### PR DESCRIPTION
**Pull request by albinantti**
Changes django admin order item view to remove filtering based on order id, and instead sort by it. This way the window doesn't become 11000 lines long, and you can get to the most recent order item quickly.
Fixes #369

### CODE REVIEW

**Comment by albinantti** [comment]
> Is it possible to repro this on dev or does one have to create like 600 orders?

We don't have any dummy data generation available. Adding some sort of automatic setup of an organisation, user, permissions and a few orders upon setting up the dev environment has been on the table for a while (at least a txt file of the commands required to create a dev user) but there's currently none afaik

**Comment by sgronlund** [comment]
> > Is it possible to repro this on dev or does one have to create like 600 orders?
> 
> We don't have any dummy data generation available. Adding some sort of automatic setup of an organisation, user, permissions and a few orders upon setting up the dev environment has been on the table for a while (at least a txt file of the commands required to create a dev user) but there's currently none afaik

Okay, that's good to know. Will do small repro locally and probably approve in a bit

File: backend/admin.py [hunk 1]
```diff
@@ -127,6 +127,7 @@ def get_menu_name(self, obj):
     def get_order_name(self, obj):
         return f'Order #{obj.order.id}'
     get_order_name.short_description = 'Order'
+    get_order_name.admin_order_field = '-order_id'
```
Comment (sgronlund) [inline backend/admin.py:130]: Should this not be just order_id? 

Locally, I get the oldest order first and if the goal is to find recent orders quickly shouldn't it be the opposite?

> First couple items
image

> Last couple items
image

**Review by sgronlund — CHANGES_REQUESTED** [review]
Small change but please confirm comment

File: backend/admin.py (hunk 1 above)
Comment (albinantti) [reply backend/admin.py]: The goal was for it to be newest first indeed, and the dash was supposed to do that but I had placed it in the wrong location. The admin_order_field tells Django how it should sort the "calculated" value of get_order_name. Added an ordering-column that orders by -order_id which does what we want it to, both for order items and for orders.

**Review by dakaza98 — APPROVED** [review]
LGTM

**Pull request merged**